src/lib.rs

```diff
@@ -85,6 +85,21 @@ impl<T> LazyCell<T> {
         unsafe { &*self.inner.get() }.as_ref()
     }
 
+    /// Borrows the contents of this lazy cell for the duration of the cell
+    /// itself
+    /// 
+    /// If the underlying value of the cell has not yet been realized, the
```

Comment:
Could you replace this sentence with: "If the cell has not yet been filled, the cell is first filled using the function provided"